docs/changes/wcs/12994.feature.rst

```diff
@@ -0,0 +1,2 @@
+Updated bundled ``WCSLIB`` to version 7.9 with several bugfixes. For a list of
```

Comment:
```suggestion
Updated bundled WCSLIB to version 7.9 with several bugfixes. For a list of
```

Reply:
Why? in the past it was recorder as \``WCSLIB\``

Reply:
Was it? If so, then just ignore.